module/python/python.go

```diff
@@ -49,12 +50,30 @@ func (i Inspector) CheckDir(dir string) bool {
 	return false
 }
 
+func parseDockerFile(dir, path string, m map[string]string) {
+	// find all PipManagerFiles from dockerfile
+	var regexpToFindPipManagerFiles = `.*?pip.*?install.*?-r[ ]*(.*?)\s+`
```

Comment:
这个正则表达式匹配的范围可能太广泛了？是不是可以考虑收紧一点，以及最后的 `\s+` 我没看懂为什么存在

Reply:
```suggestion
	var regexpToFindPipManagerFiles = `pip\d?\s+install.+?-r\s+([^\s&|;"']+)`
```
那么你看这样如何，
- 如果要求 `\s+` 会导致最后一个命令是 `pip install` 时无法匹配；
- 开头的 `.*?` 似乎是不必要的；
- `.*?` 这种写法会导致接受 `pipinstall` 和 `install-r` 这种不正确的文本
- 尽量减少非贪婪匹配（我没有经过测试，但经验上似乎这种性能更好一点，在源串足够长时

参考：https://regex101.com/r/CV38Gs/1

Reply:
如果你觉得这没有问题，可以点击commit提交这个
建议的修改